bindings_ffi/src/mls.rs

```diff
@@ -7059,4 +7089,131 @@ mod tests {
             assert_eq!(decoded.url, original.url);
         }
     }
+
+    #[tokio::test]
+    async fn test_create_dm_performance() {
+        let alix_client = new_dev_test_client().await;
+        let bo_client = new_dev_test_client().await;
+
+        let start = Instant::now();
+        let _dm = alix_client
+            .conversations()
+            .find_or_create_dm(
+                bo_client.account_address.clone(),
+                FfiCreateDMOptions::default(),
+            )
+            .await
+            .expect("DM creation should succeed");
+        let elapsed = start.elapsed();
+
+        println!("created a DM in: {}ms", elapsed.as_millis());
+        assert!(
+            elapsed < Duration::from_millis(200),
```

Comment:
We can increase this number to be something more reasonable but have it low for now cause this is what passes on Local.